mongodb-extension/pom.xml

```diff
@@ -44,5 +44,17 @@
             <version>${project.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.google.auto.service</groupId>
```

Comment:
You're using reflection.
Why not use the ClassMappings?
https://github.com/eclipse/jnosql/blob/master/mapping/mapping-core/src/main/java/org/eclipse/jnosql/mapping/reflection/ClassMapping.java

You can inject the ClassMappings:

https://github.com/eclipse/jnosql/blob/master/mapping/mapping-core/src/main/java/org/eclipse/jnosql/mapping/reflection/ClassMappings.java

Reply:
I don't understand, are you talking about Metamodel generation ? Am I using reflection in MetamodelProcessor.
Why are you saying I am using reflection while highlighting Google Autoservice ?

https://github.com/google/auto/tree/master/service

I looked at their source code, I didn't notice reflection there.

Also, annotation processors are triggered at compile time (so they usually don't cause performance issues), are you sure I can use ClassMappings ?

Reply:
Hum, I got it.